### ISSUE TITLE
Start background process with systemd-run that still logs to the current TTY

### ISSUE THREAD

**Issue by daandemeyer** [systemd/systemd#32725]
### Component

systemd-run

### Is your feature request related to a problem? Please describe

I want to start and run `bpftrace` in an integration test as a background process. To make sure bpftrace can register its probes, I am adding sd-notify to `bpftrace` so it can report when its probes are registered. I then want to run bpftrace with systemd-run which will wait until bpftrace has been started properly:

```
systemd-run --unit bpftrace --service-type=notify --pipe bpftrace <script>
```

Specifically, I don't specify `--wait` here because I want to run bpftrace in the background. Yet `--pipe` implies `--wait` anyway.

### Describe the solution you'd like

A way for systemd-run to start a service with `--pipe` or `--pty` without implying `--wait`.

### The systemd version you checked that didn't have the feature you are asking for

main

**Comment by poettering** [comment]
it might be ok to add something like that, but I am very sure that it should not be built around --pipe, but something else, since --pipe connects *both* stdin and stdout directly, and here it seems you only want stdout, right?

**Comment by DaanDeMeyer** [comment]
> it might be ok to add something like that, but I am very sure that it should not be built around --pipe, but something else, since --pipe connects _both_ stdin and stdout directly, and here it seems you only want stdout, right?

Yeah stdin doesn't matter, I just want the output on the PTY/stdout without implying --wait
